docker.make

```diff
@@ -1,10 +1,8 @@
 # This makefile is executed from inside the docker container.
```

Comment:
We can probably nix this make file and turn it into bash scripts. That's how we do it in Socorro. This make file doesn't have any dependencies for its targets, so it's not getting anything out of being a make file.